src/styles/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    margin-right: 4px;
+    background-position: center;
+    background-image: url(/src/images/star.svg);
+  }
```

Comment:
You have hard coded the star dimensions and image in the stars class. This is a code quality issue. It would be better to create a mixin that takes the size and image as parameters. This way, you can easily reuse and modify the mixin for different star sizes and images.